examples/template.html

```diff
@@ -19,117 +19,29 @@
         width: 100vw;
         margin: 0;
       }
+      #{{ plot_id }} {
```

Comment:
This is needed to get the resizing to work properly. Without it, the height stays at 500px regardless of what we do.